src/test/java/core/basesyntax/service/RegistrationServiceTest.java

```diff
@@ -0,0 +1,135 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceTest {
+    public static final String NULL_VALUE = null;
+    public static final String VALID_NAME = "Tom123";
+    public static final String VALID_PASSWORD = "aBc321";
+    public static final Integer VALID_AGE = 22;
+
+    private static RegistrationService registrationService;
+    private static User actualUser;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        actualUser = new User();
+    }
+
+    @Test
+    void userNotNullCheck() {
+        actualUser.setLogin(NULL_VALUE);
+        actualUser.setPassword(VALID_PASSWORD);
+        actualUser.setAge(VALID_AGE);
```

Comment:
also, we can add a constructor to the User class instead of initializing with setters

Reply:
same for all